Parser/spark.py

```diff
@@ -418,10 +423,10 @@ def makeState(self, state, sym):
         #  need to know the entire set of predicted nonterminals
         #  to do this without accidentally duplicating states.
         #
-        core = predicted.keys()
+        core = list(predicted.keys())
```

Comment:
I think this can be written just as
`tcore = tuple(sorted(predicted))`

Reply:
Thanks. Fixed in aef0c885e404c0165a89e2cff1920d6fd4440705.